### DISCUSSION TITLE
雷电9内存相关疑问

### DISCUSSION THREAD

**Discussion by xiaoweiXX** [MaaAssistantArknights/MaaAssistantArknights#10750] [Q&A]
如果我没记错的话，在雷电9模拟器每次截图会增加一点点已提交的内存（参考https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/3027 ），长时间运行特别是运行肉鸽这类需要大量截图的操作时会尤为严重，因为这个bug到现在我也一直没用过雷电9。而现在MAA更新了对雷电模拟器的截图增强功能，还指定要求使用雷电9，我想问一下现在这个bug解决了吗？

**Comment by ABA2396** [comment]
现在没怎么听说过有人反馈这个， ~~也可能是没人看提交内存~~  mumu 和 雷电 都支持了截图增强，可以自己试试哪个好用

**Reply by xiaoweiXX** [reply]
> 现在没怎么听说过有人反馈这个， ~~也可能是没人看提交内存~~  mumu 和 雷电 都支持了截图增强，可以自己试试哪个好用

mumu我用着不顺手，而且经常有莫名其妙的小卡顿。雷电也是小毛病一堆，一开始IP地址不断变动，换了雷电9之后刷肉鸽提交内存爆炸我光速回雷电4，后来又升级到雷电5。
提交内存爆炸之后对运行也是有很大影响的，首先是占硬盘，如果我没记错Windows设置默认的虚拟内存是无上限的；如果手动设置限制虚拟内存大小或者硬盘被虚拟内存占满了之后，随着虚拟内存慢慢被占满，模拟器的运行也会越来越卡，直到最后闪退，就完全没法用了。
测试这个也很麻烦，得刷肉鸽刷一会才能看出bug还在不在。两年前发现这个问题我就给雷电提交反馈了，一直也不知道这个修没修，雷电好像也没个回复机制。
